topology/probes/fabric.go

```diff
@@ -162,10 +162,16 @@ func NewFabricProbe(g *graph.Graph) *FabricProbe {
 
 	list := config.GetConfig().GetStringSlice("analyzer.topology.fabric")
 	for _, link := range list {
-		pc := strings.Split(link, "->")
-		if len(pc) != 2 {
-			logging.GetLogger().Errorf("FabricProbe link definition should have two endpoint: %s", link)
-			continue
+		l2OnlyLink := false
+		pc := strings.Split(link, "-->")
```

Comment:
could be nice to add an example in the default config file

Reply:
ack